custom_typings/proxy.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'proxy' {
```

Comment:
what is this custom_typing supposed to be? I don't see a `proxy` package in our package.json.